#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_maximize.patch  <meskes@debian.org>
##
## DP: Change default configuration to NOT start maximized.
--- tora-1.3.21pre22.orig/src/tomain.cpp
+++ tora-1.3.21pre22/src/tomain.cpp
@@ -454,7 +454,7 @@
         TOCATCH
     }
 
-    if (!toConfigurationSingle::Instance().globalConfig(CONF_MAXIMIZE_MAIN, "Yes").isEmpty() && Connections.empty() )
+    if (!toConfigurationSingle::Instance().globalConfig(CONF_MAXIMIZE_MAIN, "").isEmpty() && Connections.empty() )
         showMaximized();
     else
         show();
--- tora-1.3.21pre22.orig/src/toglobalsetting.cpp
+++ tora-1.3.21pre22/src/toglobalsetting.cpp
@@ -84,7 +84,7 @@
     HistorySize->setValue(toConfigurationSingle::Instance().globalConfig(CONF_STATUS_SAVE,
                           DEFAULT_STATUS_SAVE).toInt());
     IncludeDB->setChecked(!toConfigurationSingle::Instance().globalConfig(CONF_DB_TITLE, "Yes").isEmpty());
-    MaximizeMain->setChecked(!toConfigurationSingle::Instance().globalConfig(CONF_MAXIMIZE_MAIN, "Yes").isEmpty());
+    MaximizeMain->setChecked(!toConfigurationSingle::Instance().globalConfig(CONF_MAXIMIZE_MAIN, "").isEmpty());
     Statusbar->setChecked(!toConfigurationSingle::Instance().globalConfig(CONF_MESSAGE_STATUSBAR, "").isEmpty());
     RestoreSession->setChecked(!toConfigurationSingle::Instance().globalConfig(CONF_RESTORE_SESSION, "").isEmpty());
     HelpDirectory->setText(toHelpPath());
